<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pharmacist</class>
 <widget class="QWidget" name="Pharmacist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_background">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-20</y>
     <width>831</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#1A1627;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="title">
    <string>Re√ßeteler</string>
   </property>
   <widget class="QTextBrowser" name="txt_allAppointment_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_Image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>141</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid;
border-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ID">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Ad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_appointmentID">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_giveMedicine">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Give Med</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
